### ISSUE TITLE
Výběr modelu k indexaci - změna obsahu roletky

### ISSUE THREAD

**Issue by kerschfilip** [ceskaexpedice/kramerius-admin-client#81]
přesunuto z #58 

Prosím o úpravu obsahu roletky s výběrem modelu k indexaci tak, aby všechny modely podávalo jádro.

![image](https://github.com/ceskaexpedice/kramerius-admin-client/assets/57805328/4dbd9d45-8e71-484d-a4db-a07956ba354d)

**Comment by kerschfilip** [comment]
Vypadá to, že teď jádro modely podává, ale některé modely nemají definovaný název - to je asi na @pkudela? 

![image](https://github.com/ceskaexpedice/kramerius-admin-client/assets/57805328/fb0edb12-96e5-44d2-9468-693ec3e68e6e)

Testováno na k7Admin - verze: 1.4.2 - commit: geaa9c90, Jádro verze: 7.0.26 - commit: cd2f8

**Comment by pkudela** [comment]
Jojo, to je asi na me, lokalizaci dodelam.

**Comment by kerschfilip** [comment]
@pkudela @pavel-stastny myslím, že stále chybí lokalizace některých hodnot:

![Image](https://github.com/user-attachments/assets/43afc85c-3e0d-424b-af46-8e31a8e5dfcf)

Zkoušel jsem teď na Klient Verze: 1.4.9 Commit: [405682f](https://github.com/ceskaexpedice/kramerius-admin-client/commit/405682f)

**Comment by pavel-stastny** [comment]
@pkudela Můžeš komitnout tu lokalizaci pro article? 

@kerschfilip Ohledně ndkmonographvolume, to je ProARC interní model, ne ?

**Comment by kerschfilip** [comment]
jo, pardon, je to tak. To tam být nemá, koukal jsme na náš test, kde je trochu bordel :-)
